storage/badger/procedure/cluster.go

```diff
@@ -14,29 +14,35 @@ import (
 
 // InsertClusterBlock inserts a cluster consensus block, updating all
 // associated indexes.
-func InsertClusterBlock(block *cluster.Block) func(*badger.Txn) error {
+func InsertClusterBlock(block *cluster.BlockProposal) func(*badger.Txn) error {
```

Comment:
```suggestion
func InsertClusterBlock(proposal *cluster.BlockProposal) func(*badger.Txn) error {
```
In most cases I think it will be clearer to rename local variables which are now `*Proposal`-typed to `proposal` or similar.